deployment/model-testing/check_model_dependencies.ipynb

```diff
@@ -0,0 +1,206 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Validate a Model's Environment Contains the Required Dependencies\n",
+    "This example demonstrates how to check that a model's environment contains all the dependencies it needs to run.\n",
+    "This allows developers and data scientists to validate the environment locally before attempting to deploy the\n",
+    "model to Verta."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Imports"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "from verta import Client\n",
+    "from verta.environment import Python\n",
+    "from verta.registry import VertaModelBase, verify_io, _check_model_dependencies as check_model_dependencies\n",
```

Comment:
Since the method is public now, I think we can do

```suggestion
    "from verta.registry import VertaModelBase, verify_io, check_model_dependencies\n",
```

Reply:
It looks like you fixed this in f74d5bb3b7d3d8693b3c0a3e3492ed9e5187988d, but then undid it in dc34759bc58b2537c32bbea1eece529df16cf7f0. Please be careful with your commits.